src/adapt/epub.js

```diff
@@ -1084,7 +1084,7 @@ adapt.epub.OPFView.prototype.finishPageContainer = function(viewItem, page, page
     } else {
         viewItem.instance.viewport.contentContainer.appendChild(page.container);
     }
-    this.pageSheetSizeReporter(viewItem.instance.pageSheetSize, viewItem.item.spineIndex, pageIndex);
+    this.pageSheetSizeReporter({ width: viewItem.instance.pageSheetWidth, height: viewItem.instance.pageSheetHeight }, viewItem.instance.pageSheetSize, viewItem.item.spineIndex, pageIndex);
```

Comment:
`pageIndex` is the page index in the spine item.
Since `adapt.viewer.Viewer.prototype.setPageSize` requires the serial page index through the entire document, `pageIndex` here should be changed to the following:
```javascript
viewItem.instance.pageNumberOffset + pageIndex
```